src/docs/setup-swagger.ts

```diff
@@ -0,0 +1,16 @@
+import type { FastifyInstance } from 'fastify';
+import fastifySwagger from 'fastify-swagger';
+
+export const setupSwagger = (server: FastifyInstance) => {
+
+  server.register(fastifySwagger, {
+    routePrefix: '/docs',
+    swagger: {
+      info: {
+        title: 'Image Service',
+        version: require('../../package.json').version,
+      },
+    },
+    exposeRoute: true,
```

Comment:
Do you want to always expose API docs or expose it for dev environments only?

Reply:
Always